src/lib/wicked/adapter.js

```diff
@@ -113,6 +113,33 @@ class WickedAdapter {
         return result;
     }
 
+    /*
+     * Obtains the DNS global settings from /etc/sysconfig/network/config
+     */
+    async dnsSettings() {
+        const filePath = `/etc/sysconfig/network/config`;
+        const file = await new SysconfigFile(filePath).read();
+        const policy = file.get("NETCONFIG_DNS_POLICY");
+        const nameServers = (file.get("NETCONFIG_DNS_STATIC_SERVERS") || "").split(" ");
```

Comment:
NP: I still feeling that we could improve this, but right now I have no idea how (I mean, without forcing to change everything related to _split_s and _join_s).

However, just a note for the future, the SysconfigFile#get method could receive the desired default value as a second param.